CMakeLists.txt

```diff
@@ -292,3 +292,7 @@ add_custom_target(clean-built
    COMMAND ${CMAKE_COMMAND} -E remove_directory ${PROJECT_SOURCE_DIR}/built
    COMMENT "Cleaning `built` directory"
 )
+
+# Debian Package
+include(cmake/CPackLinux.cmake)
```

Comment:
In the same vein as the earlier KISS comment, we can consider simply `cmake/packaging.cmake`

There's often significant overlap (description, maintainer, etc, etc) that the if/else's in a shared file is less onerous than dealing with split files and some common properties when supporting multiple packages.